go.mod

```diff
@@ -45,6 +45,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.16.3
 )
 
+replace github.com/aquasecurity/tracee/api => github.com/rscampos/tracee/api v0.0.0-20240520202449-d199291ce929
```

Comment:
don't forget to split this pr into 2, go.mod enters api first, main go.mod follows it.

Reply:
@geyslan splited the PR into 2, if you don't mind to do the last review, tks again :)